### PR TITLE
Add svg to page and update surrounding content

### LINKED ISSUE #8652: Issue with "Introduction": Images in tutorial are not SVGs
MDN URL: https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Introduction

#### What information was incorrect, unhelpful, or incomplete?

> SVG is an XML language, similar to XHTML, which can be used to draw vector graphics, such as the ones shown to the right.

The image to the right is a png, a raster graphics format. 

#### Specific section or headline?

Introduction

#### What did you expect to see?

The SVG tutorial should contain examples of fully fleshed out SVGs, not raster screenshots of them.

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/svg/tutorial/introduction`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Introduction
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/svg/tutorial/introduction/index.html
* Last commit: https://github.com/mdn/content/commit/4a810357cb7f1a75de023c65fc0c83a884bd2b08
* Document last modified: 2021-03-25T00:07:10.000Z

</details>

### LINKED ISSUE #11389: Issue with "Introduction": Text references missing images.
MDN URL: https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Introduction

#### What information was incorrect, unhelpful, or incomplete?

The first sentence references "... [svg graphics] shown to the right." These graphics do not appear to exist anymore. To fix, consider removing the reference from the text, adding graphics, or referencing an existing svg like the site logo.

#### Specific section or headline?

Under the h1 "Introduction."

#### What did you expect to see?

Sample images to the right of the text.

#### Did you test this? If so, how?

Viewed the page in Firefox.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/svg/tutorial/introduction`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Introduction
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/svg/tutorial/introduction/index.md
* Last commit: https://github.com/mdn/content/commit/bd3f490ef78b600a1435dfe3611a27f3f2fbf4ef
* Document last modified: 2021-09-23T21:22:12.000Z

</details>

### PR DESCRIPTION
**Pull request by AnilSeervi**
#### Summary

- Add dino svg : Previously a `png` image was displayed to the right of the text.
- Update surrounding content : images can't be made to float right in markdown syntax
#### Motivation

Open-source
#### Related issues

Fixes #8652 
Closes #11389
#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/svg/tutorial/introduction/index.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ tags:
 ---
 {{ PreviousNext("Web/SVG/Tutorial", "Web/SVG/Tutorial/Getting_Started") }}
 
-[SVG](/en-US/docs/Web/SVG) is an [XML](/en-US/docs/Web/XML) language, similar to [XHTML](/en-US/docs/Glossary/XHTML), which can be used to draw vector graphics, such as the ones shown to the right. It can be used to create an image either by specifying all the lines and shapes necessary, by modifying already existing raster images, or by a combination of both. The image and its components can also be transformed, composited together, or filtered to change their appearance completely.
+[SVG](/en-US/docs/Web/SVG) is an [XML](/en-US/docs/Web/XML) language, similar to [XHTML](/en-US/docs/Glossary/XHTML), which can be used to draw vector graphics, such as the one shown below. It can be used to create an image either by specifying all the lines and shapes necessary, by modifying already existing raster images, or by a combination of both. The image and its components can also be transformed, composited together, or filtered to change their appearance completely.
+
+![](dino.svg)
```
Comment (wbamberg) [inline files/en-us/web/svg/tutorial/introduction/index.md:17]:
```suggestion
![Mozilla dino logo](dino.svg)
```

Comment (AnilSeervi) [reply files/en-us/web/svg/tutorial/introduction/index.md:17]: Yes.

File: files/en-us/web/svg/tutorial/introduction/index.md [hunk 2]
```diff
@@ -12,7 +12,9 @@ tags:
 ---
 {{ PreviousNext("Web/SVG/Tutorial", "Web/SVG/Tutorial/Getting_Started") }}
 
-[SVG](/en-US/docs/Web/SVG) is an [XML](/en-US/docs/Web/XML) language, similar to [XHTML](/en-US/docs/Glossary/XHTML), which can be used to draw vector graphics, such as the ones shown to the right. It can be used to create an image either by specifying all the lines and shapes necessary, by modifying already existing raster images, or by a combination of both. The image and its components can also be transformed, composited together, or filtered to change their appearance completely.
+[SVG](/en-US/docs/Web/SVG) is an [XML](/en-US/docs/Web/XML) language, similar to [XHTML](/en-US/docs/Glossary/XHTML), which can be used to draw vector graphics, such as the one shown below. It can be used to create an image either by specifying all the lines and shapes necessary, by modifying already existing raster images, or by a combination of both. The image and its components can also be transformed, composited together, or filtered to change their appearance completely.
+
+![Moziall dino logo](dino.svg)
```
Comment (wbamberg) [inline files/en-us/web/svg/tutorial/introduction/index.md:17]:
```suggestion
![Mozilla dino logo](dino.svg)
```

(typo fix)

**Review by wbamberg — APPROVED** [review]
👍 thank you @AnilSeervi !

**Pull request merged**